frontend/README.md

```diff
@@ -1,36 +1,37 @@
-- old component styles use BEM notation (at least it should): `block__element_modifier`. Also there are `mix` classes: `block_modifier`.
-- new way to naming CSS selectors is camel-case like `blockElemenModifier` and use `classnames` to combine it
-- component base style resides in the component's root directory with name of component converted to kebab-case. For example `ListComments` style is located in `./app/components/list-comments/list-component.tsx`
-- any other files should be named also in kebab-case. For example `./app/utils/get-param.ts`
+* now we are migrating to CSS Modules and this is a recommended way to stylization. A file with styles should be named like `component.module.css`
+* old component styles use BEM notation (at least it should): `block__element_modifier`. Also, there are `mix` classes: `block_modifier`
+* new way to naming CSS selectors is camel-case like `blockElemenModifier` and use `classnames` to combine it
+* component base style resides in the component's root directory with a name of component converted to kebab-case. For example, `ListComments` style is located in `./app/components/list-comments/list-component.tsx`
+* any other files should be named also in kebab-case. For example, `./app/utils/get-param.ts`
 
 ### Imports
 
-- imports for typescript, javascript files should be without extension: `./index`, not `./index.ts`
-- if file resides in the same directory or in subdirectory import should be relative: `./types/something`
-- otherwise it should be imported by absolute path relative to `src` folder like `common/store` which mapped to `./app/common/store.ts` in webpack, tsconfig and jest
+* imports for TypeScript, JavaScript files should be without extension: `./index`, not `./index.ts`
+* if the file resides in the same directory or subdirectory import should be relative: `./types/something`
+* otherwise it should be imported by absolute path relative to `src` folder like `common/store` which mapped to `./app/common/store.ts` in webpack, tsconfig, and Jest
 
 ### Testing
 
-- project uses `jest` as test harness.
-- jest check files that match regex `\.(test|spec)\.ts(x?)$`, i.e `comment.test.tsx`, `comment.spec.ts`
-- tests are running on push attempt
-- example tests can be found in `./app/store/user/reducers.test.ts`, `./app/components/auth-panel/auth-panel.test.tsx`
+* project uses `jest` as test harness
+* Jest checks files that match regex `\.(test|spec)\.ts(x?)$`, i.e., `comment.test.tsx`, `comment.spec.ts`
+* tests are running on push attempt
+* example tests can be found in `./app/store/user/reducers.test.ts`, `./app/components/auth-panel/auth-panel.test.tsx`
 
 ### How to add new locale
 
-Please see [this documentation](https://github.com/umputun/remark42/blob/master/docs/translation.md).
+Please see [this documentation](https://remark42.com/docs/latest/translation/).
```

Comment:
Links to MD documentation files are intentional, please don't replace them with links to the site yet. What should be done is replacing them with the correct location in the [new docs directory](https://github.com/umputun/remark42/tree/master/site/src/docs).

Reply:
Fixed.